src/main/java/com/synopsys/integration/detect/lifecycle/run/operation/OperationRunner.java

```diff
@@ -1442,4 +1452,26 @@ public UUID getScanIdFromScanUrl(HttpUrl blackDuckScanUrl) {
     public DetectConfigurationFactory getDetectConfigurationFactory() {
         return this.detectConfigurationFactory;
     }
+
+    private BinaryUploader createMultipartBinaryScanUploader(File binaryUpload, NameVersion projectNameVersion,
```

Comment:
It might be good to move this method someplace else (some factory perhaps) to not make the OperationRunner class even larger.